### ISSUE TITLE
Missing condition in section 3.3.8, first "f."

### ISSUE THREAD

**Issue by emuller-amazon** [w3c/jlreq#15]
Section 3.3.8, first set of a., b., c., etc. 

f. When the adjacent character is one of the closing brackets (cl-02), the ruby text may...
...
h. Also, when the adjacent character is one of the opening brackets (cl-01) before the ruby object, the ruby text may...

f. fails to say this is only when the bracket is _after the ruby object_ h. does indicate this corresponding restriction (_before the ruby object_)

So, at the very least, insert "after the ruby object" after "(cl-02)" in f.

Even better, I think that f, g and h can be combined:

When the preceding character is an opening bracket, or when the following character is a closing bracket, a comma or a full stop, the ruby text may go over these adjacent characters, up to the full-width size of a ruby character. Note that the overhang must not go beyond those adjacent characters.

**Comment by himorin** [comment]
Toshi Kobayashi san updated text of f,g,h by combining into one.

**Comment by macnmm** [comment]
@himorin I have some edits to make over the entire ruby section's English. Should we close this issue out first, or should I address the whole section as well as the few lines involved in this issue?

**Comment by himorin** [comment]
I'd want to have an issue for edits over the entire this section first, also I don't think closing this without fix is a good way.

**Comment by macnmm** [comment]
OK I was able to separate out the fix for #10. However in re-wording the fix for #15, I changed "ruby letter" to "ruby character", and "go over" to "overhang". This introduced inconsistencies in all the places where the former phrases are used. So, I unified on the phrases I prefer. That is why the #15 change covers more areas. So, this time I can use the "letter" and "go over" phrasing in my smaller change and create an issue about "letter" and "go over" to clean up further in another PR...
